node/pkg/alephium/watcher.go

```diff
@@ -478,17 +482,29 @@ func isEventConfirmed(
 	eventBlockHeader *sdk.BlockHeaderEntry,
 	currentTs int64,
 	currentHeight int32,
+	isMainnet bool,
 ) bool {
 	consistencyLevel := event.msg.consistencyLevel
 	if eventBlockHeader.Height+int32(consistencyLevel) > currentHeight {
 		logger.Debug("not enough block confirmations", zap.String("txId", event.TxId), zap.Uint8("consistencyLevel", consistencyLevel))
 		return false
 	}
 
-	duration := int64(consistencyLevel) * BlockTimeMs
+	duration := getConfirmationDuration(isMainnet, event.msg.IsTransferTokenVAA(), consistencyLevel)
 	if eventBlockHeader.Timestamp+duration > currentTs {
 		logger.Debug("not enough confirmation time", zap.String("txId", event.TxId), zap.Uint8("consistencyLevel", consistencyLevel))
 		return false
 	}
 	return true
 }
+
+func getConfirmationDuration(
+	isMainnet bool,
+	isTransferTokenVAA bool,
+	eventConsistencyLevel uint8,
+) int64 {
+	if isMainnet && isTransferTokenVAA {
+		return int64(maxUint8(eventConsistencyLevel, MinimalConsistencyLevel)) * BlockTimeMs
```

Comment:
Since the current `minimalConsistencyLevel` in the contract is 105, and after the Rhone upgrade, the maximum reorg size is 200, we need to wait for at least 205 * 16s. Do you think we need to update the `minimalConsistencyLevel` in the contract? This would require each guardian to sign the governance VAA.